### ISSUE TITLE
Add tips to status bar

### ISSUE THREAD

**Issue by imsnif** [zellij-org/zellij#938]
In a recent release, we added the ability to show more tips in the status bar. Right now, the default tip is the quicknav tip:

`Tip: Alt + <n> => open new pane. Alt + <[] or hjkl> => navigate between panes. Alt + <+-> increase/decrease pane size`

I'd like to collect suggestions to add more tips and record them (and their implementation) here. Please offer more suggestions by commenting on this issue and I'd be happy to add them:

[ ] - `Tip: Send mouse clicks directly to your terminal to mark text or click on links by holding SHIFT while clicking with the mouse.`

**Comment by casonadams** [comment]
would be nice to disable the tips "second_line" all together

**Comment by imsnif** [comment]
> would be nice to disable the tips "second_line" all together

Agreed, but I feel this is a separate issue. Maybe as a config option?

**Comment by auronandace** [comment]
Not sure if this would count as a tip or a feature request. Perhaps some sort of indicator to show caps lock, scroll lock or num lock is active.

**Comment by raphCode** [comment]
I have comments on when/how to display tips:

[gitui](https://github.com/extrawurst/gitui) has an extra key to toggle between display of all currently available keybindings/tips and those who fit into a single line.

One could also hide tips the user already uses often, to make room for new, yet unknown tips.

**Comment by a-kenji** [comment]
Maybe these tips here in this screenshot should be distinct and not been shown at the same time?

![image](https://user-images.githubusercontent.com/65275785/151855954-86ca51d3-1c78-4a34-87e5-e97a918ba305.png)

This feels very busy to me personally and might detract from the usefulness.

**Comment by raphCode** [comment]
- The zellij shortcuts like `a` for attach or `ls` like list-sessions were quite hard to find for me personally. (They do not even show up in `zellij --help`) Maybe this would be a great candidate for a tip?
- Also, if the amount of plugins increases in the future, we can feature some builtin / popular plugins and how to activate / use them?
- Same goes for sessions, to make users aware that they exist and what they can do with it.

**Comment by raphCode** [comment]
Since someone asked in discord what `Sync` does, and it was new for me as well:
"The sync function of a tab sends the keyboard input to all its panes simultaneously"

**Comment by raphCode** [comment]
I just discovered that floating panes can be moved with the mouse, this might be worth a tip as well

**Comment by har7an** [comment]
Since we already have a bunch of tips now, can this be closed?

**Comment by imsnif** [comment]
Yes, I think we can keep adding them on the fly. I can think of a few new ones with the latest release.
